docs/pages/cloud/downloads.mdx

```diff
@@ -51,6 +51,33 @@ Linux Debian Packages              | [amd64](https://get.gravitational.com/telep
 Linux RPM Packages                 | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.x86_64.rpm) | [arm64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm64.rpm) | [armv7](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm.rpm) | [i386](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.i386.rpm) |
 macOS Packages                     | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=).pkg) |
 
+## Teleport package repositories
+
+Teleport currently maintains DEB repositories for Debian compatible systems.
```

Comment:
I'm guessing we don't want to share RPM instructions because RPM repo is not available yet?

Reply:
Yep. We'll add it here when we have properly versioned RPM repos (likely next week).